### PR TITLE
improve example group_vars

### PR DESCRIPTION
**Pull request by przemeklal**
- Add missing dictionary for example net-attach-def creations.
- Rework proxy settings to be more Kubespray-like.

### CODE REVIEW

**Comment by przemeklal** [comment]
@michaelspedersen @lmdaly please take a look

File: examples/group_vars/all.yml [hunk 1]
```diff
@@ -32,10 +32,19 @@ gpu_dp_namespace: kube-system
 # Forces installation of the Multus CNI from the official Github repo on top of the Kubespray built-in one
 force_external_multus_installation: true
 
+# Create reference net-attach-def objects
+example_net_attach_defs:
```
Comment (michaelspedersen) [inline examples/group_vars/all.yml:36]: Consider changing the default to false, as I imagine it can cause issues if the userspace parts are not installed

Comment (przemeklal) [reply examples/group_vars/all.yml:36]: actually net-attach-def won't do anything as long as the user doesn't reference it in their pod spec, but yea, it makes sense to not create them by default

**Comment by michaelspedersen** [comment]
Looks good to me

**Pull request merged**